packages/xrpl/test/models/MPTokenIssuanceSet.test.ts

```diff
@@ -55,7 +55,7 @@ describe('MPTokenIssuanceSet', function () {
 
   it(`throws w/ conflicting flags`, function () {
     /* eslint-disable no-bitwise -- Bitwise operation needed for flag combination */
-    const invalid = {
+    let invalid = {
       TransactionType: 'MPTokenIssuanceSet',
       Account: 'rWYkbWkCeg8dP6rXALnjgZSjjLyih5NXm',
       MPTokenIssuanceID: TOKEN_ID,
```

Comment:
Instead of redefining every test, might be easier to follow the pattern used in other test files (define the object in the beforeEach runner, and then selectively set missing fields to undefined or adding to the object over each test)